src/firewall/firewalld.rs

```diff
@@ -373,6 +374,16 @@ impl firewall::FirewallDriver for FirewallD {
             setup_portfw.container_id
         );
 
+        if let Some(port_mappings) = setup_portfw.port_mappings {
```

Comment:
This is duplicated for every driver doing the exact same, would it not be better to de duplicate this by moving it into the  setup_firewall() and teardown_firewall() functions in bridge.rs?

That way we don't forget if ever a new driver is added.